scripts/install/run.sh

```diff
@@ -299,54 +289,55 @@ function stop_apps {
 
 function run_e2e_tests {
     printh "Running e2e tests on app"
-    pushd $E2E_TEST_DIR > /dev/null
+    pushd ${E2E_TEST_DIR} > /dev/null
     yarn
     popd > /dev/null
-    pushd $CLONE_DIR > /dev/null
+    pushd ${CLONE_DIR} > /dev/null
     yarn e2e:cy:run
     popd > /dev/null
 }
 
 function cmd_help {
     echo "Usage: run [command]"
     echo "Available commands are:"
-    echo " install"
-    echo " install_npm"
+    echo " install (from sources)"
+    echo " install_npm (from latest npm packages)"
     echo " start"
     echo " stop"
     echo " e2e"
     echo " help"
 }
 
-if [ -z "$1" ]; then
+if [ -z "${1}" ]; then
     cmd_help
     exit 1
 fi
 
-readonly commands="$1"
+readonly commands="${1}"
 
-echo "Loading configs from ./config.default.sh"
-    . ./config.default.sh
```

Comment:
this change caused the default configuration is not loaded (only `config.sh` file is being loaded)